changelog.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 0.7.0
+
+- Sass build plugin: (_breaking change_) Switched from `node-sass` to `dart-sass` for Sass compilation
+- App context package: (_breaking change_) Refactored to throw an error when unrecognised properties are added
+- Base styles UI component: Added a new package which centralises all of our global base styles including basic typography, grid, and font loading.
+- Shell UI component: Refactored to avoid output of additional `<div>` elements when providing React components as contents
+- (_breaking change_) Removed the CLI package
+- (_breaking change_) Removed the Pluggable package
+
```

Comment:
add something for:
```suggestion
- Refactor `dotcom-build-` packages into Webpack plugins
- Add `dotcom-build-base` package
```
?

Reply:
Done, didn't realise this was a suggestion so could have saved myself the effort!